File: Java/src/main/java/org/sisuns/condicionales/basico/Ejercicio_04.java
package org.sisuns.condicionales.basico;

public class Ejercicio_04 {
    public static void main(String[] args) {
        // Edad predeterminada para el ejemplo
        int edad = 25;

        // Determinar en qué rango se encuentra
        if (edad >= 0 && edad <= 12) {
            System.out.println("Niño");
        } else if (edad >= 13 && edad <= 17) {
            System.out.println("Adolescente");
        } else if (edad >= 18 && edad <= 64) {
            System.out.println("Adulto");
        } else {
            System.out.println("Adulto Mayor");
        }
    }
}

File: Python/Condicional/Basico/Ejercicio_04.py
# Solicitar al usuario ingresar su edad
edad = int(input("Ingrese su edad: "))

# Determinar en qué rango se encuentra
if 0 <= edad <= 12:
    print("Niño")
elif 13 <= edad <= 17:
    print("Adolescente")
elif 18 <= edad <= 64:
    print("Adulto")
else:
    print("Adulto Mayor")
